bankingapp/apps/netbankingapp/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.urls import path
+from apps.netbankingapp.views import UserViewSet, AccountViewSet, TransactionViewSet
+from . import views
+
+urlpatterns = [
+    path('', views.home, name='home'),
+    path('create-user/', UserViewSet.as_view({'put': 'create_user'})),
```

Comment:
Why is the URL naming convention different for creation of users and other user related stuff